sql/updates/characters/2015_08_26_00_characters_335.sql

```diff
@@ -0,0 +1,16 @@
+-- Keep only the highest guid PvE or PvP (not bones) corpse per player guid
+DELETE c FROM `corpse` c LEFT JOIN
+(
+    SELECT MAX(`corpseGuid`) AS id
```

Comment:
I just disabled that to get it working, but this still needs to be fixed.
Everyone who updates a fresh install from now on will fail at this line. Someday everyone has to reinstall and update I guess.